invenio_users_resources/views.py

```diff
@@ -11,6 +11,27 @@
 
 from flask import Blueprint
 
+blueprint = Blueprint(
+    "invenio_users_resources",
+    __name__,
+    template_folder="templates",
+    static_folder="static",
+)
+
+
+@blueprint.record_once
+def init(state):
+    """Init app."""
+    app = state.app
+    # Register services - cannot be done in extension because
+    # Invenio-Records-Resources might not have been initialized.
+    rr_ext = app.extensions["invenio-records-resources"]
+    ext = app.extensions["invenio-users-resources"]
+
+    # services
+    rr_ext.registry.register(ext.users_service)
+    rr_ext.registry.register(ext.groups_service)
+
```

Comment:
comment: both services are using the `indexer` in the rebuild_index. Might be a good idea to register those?

```suggestion
    idx_ext = app.extensions["invenio-indexer"]

     # services
     idx_ext.registry.register(ext.users_service.indexer, indexer_id="users")
     idx_ext.registry.register(ext.groups_service.indexer, indexer_id="groups")

```

Reply:
Done, thanks a lot for the suggestion!